ui/app/abilities/role.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import AbstractAbility from './abstract';
```

Comment:
This is a `ability` file in Ember. This lets use use syntax like `{{#if can "delete role"}}`, etc. elsewhere. In the case of roles/tokens/policies, all abilities are restricted to management tokens, but this may not always be the case.